styles/designs/cosmos/parts/_keyterms-components.scss

```diff
@@ -0,0 +1,34 @@
+$KeyTerms__Container: (
+    _name: "KeyTermsContainer",
+    _subselector: "",
+    _properties: (
+        margin-bottom: v-spacing(2),
+    ),
+);
+
+$KeyTerms__List:(
+    _name: "KeyTermsList",
+    _subselector: " > dl",
+    _properties:(
+        text-indent: h-spacing(-3),
+        padding-left: h-spacing(3),
+    ),
+);
+
+$KeyTerms__Term:(
+    _name: "KeyTermsTerm",
+    _subselector: " > dt ",
```

Comment:
@wilkus27 It is small thing - there is additional space added at the end of ` > dt` selector. Same thing in the below `KeyTermsDefinition` component.

Reply:
Nice catch :wink:

Reply:
It doesn't have an impact on styles, so let me fix that in my second PR for EOC refactoring (https://github.com/openstax/ce-styles/pull/8)